ensembl_metadata/urls/assembly.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path, re_path
```

Comment:
I'm not quite sure I understand this - in the `metadata_registry/urls.py` file, in urlpatterns we have: `path('assembly/', include('ensembl_metadata.urls.assembly'))`. I don't think we would want to use `include ('ensembl_metadata.urls')`, because the path would need to be different for each set of urls. This is how the urls were structured before, but would it be better practice to have the 'assembly' path in every urlpattern in `urls/assembly.py`, and not in `metadata_registry/urls.py`?

Reply:
Do not bother too much for now, the include is only there to set an initial sub route, then having all related urls in one single files is "good practice" but not mandatory.